viewer/src/main/webapp/viewer-html/components/components.json

```diff
@@ -618,6 +618,18 @@
         "singleton": true,
         "showHelp": true
     },
+    {
+        "className": "viewer.components.EditBulk",
+        "name": "Edit Bulk",
+        "shortName": "EdB",
+        "group": "Component",
+        "sources": ["Edit.js", "EditBulk.js","CurrentLocation.js"],
+        "configSource": ["ConfigObject.js","SelectionWindowConfig.js","Edit-config.js"],
```

Comment:
maybe you want to have your own `BulkEdit-config` so you can force some of the config options mapping to buttons to `false` eg. copy, new,...

Reply:
added EditBulk-config.js